src/components/input-dropdown/input-dropdown.component.ts

```diff
@@ -60,14 +61,18 @@ export class InputDropdownComponent<T> implements ControlValueAccessor, OnChange
 
     ngOnChanges(changes: SimpleChanges): void {
         if (changes.selected) {
-            if (this.menuTrigger) {
+            if (this.menuTrigger && this._viewInitialized) {
```

Comment:
Instead of having to do this view initialized thing, is it possible to use the `firstChanges` property to skip this initial close menu call, eg:

```typescript
if (this.menuTrigger && !changes.selected.firstChange)
```

This might not work, but it would be great to check if it did

Reply:
I tried it, but unfortunately, it does not always work. Sometimes the first selection click in the component does not close the menu.

Reply:
However, I liked the idea so much that I played around with it some more. I found and fixed a defect in our component and now it works. I just committed both fixes.

Reply:
Great! Thanks for making the changes!